ui/constants/action_types.js

```diff
@@ -114,6 +114,7 @@ export const UPDATE_SEARCH_SUGGESTIONS = 'UPDATE_SEARCH_SUGGESTIONS';
 export const DAEMON_SETTINGS_RECEIVED = 'DAEMON_SETTINGS_RECEIVED';
 export const CLIENT_SETTING_CHANGED = 'CLIENT_SETTING_CHANGED';
 export const UPDATE_IS_NIGHT = 'UPDATE_IS_NIGHT';
+export const WALLET_SERVERS_CACHED = 'WALLET_SERVERS_CACHED';
```

Comment:
Probably `save` instead of `cache` since we are gonna keep these around?

Either way `CACHE_WALLET_SERVERS` or `SAVE_WALLET_SERVERS`

Are these custom servers? If so, then `SAVE_CUSTOM_WALLET_SERVERS`